htlcswitch/circuit_map_test.go

```diff
@@ -120,7 +125,7 @@ func TestCircuitMapCleanClosedChannels(t *testing.T) {
 				{InKey: inKey20, OutKey: outKey20},
 			},
 			deleted: []htlcswitch.Keystone{
-				{InKey: inKey00}, {InKey: inKey11},
+				{InKey: inKey10}, {InKey: inKey11},
```

Comment:
👍

Reply:
I think I'm missing some context here?

Reply:
So the test case wants to test we delete the half-open circuits based on their `inKey`. In this case, `inKey10` and `inKey11` should be deleted. I made the mistake of putting down `inKey00`, yet the test case still passed due to how we validated the deletion - by checking the specified key couldn't be found, and `inKey00` surely couldn't be found.